debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -995,6 +995,15 @@ public static AutoCreateMode parse(String value, String defaultValue) {
                     "the original value is a toasted value not provided by the database. " +
                     "If starts with 'hex:' prefix it is expected that the rest of the string repesents hexadecimally encoded octets.");
 
+    public static final Field MONEY_FRACTION_DIGITS = Field.create("money.fraction.digits")
```

Comment:
On this setting it's worth point out that this may fall a bit short in complex scenarios. You may have different columns with different currencies requiring different fraction digits. But I don't think we need to address this right now. This column type isn't used that much to begin with, this requirement seems really special, and we can adjust the format of the option value to accept a value-per-column format down the road (we do have that for other options already). I.e. no need to change this right now, just mentioning this for the sake of completeness.